src/lib/y2storage/proposal_settings.rb

```diff
@@ -448,6 +587,51 @@ def legacy_check_root_snapshots
       root_filesystem_type.is?(:btrfs) && use_snapshots
     end
 
+    # Implementation for {#volumes_sets} when {#separate_vgs} is set to false
+    #
+    # @see #volumes_sets
+    #
+    # @return [Array<VolumeSpecificationsSet]
+    def vol_sets_plain
+      if lvm
+        [VolumeSpecificationsSet.new(volumes.dup, :lvm)]
```

Comment:
The [`VolumeSpecificationsSet` has a `#push` method](https://github.com/yast/yast-storage-ng/pull/936/files/234256464c9b135737fec7d4a3a9fc7ad454dbcc#diff-95853e2aee6fe6cac00c9d2d14872012R81) which includes a received volumes into its `volumes` attribute. We want to avoid that this operation could modify the original `volumes`.

Reply:
Ok, thanks for the explanation ;)